src/DetailsView/Styles/detailsview.scss

```diff
@@ -304,101 +304,65 @@ div.insights-file-issue-details-dialog-container {
                 }
             }
         }
-        .test-steps-nav-container {
-            display: flex;
-            overflow-y: auto;
-            overflow-x: hidden;
-            border-right: 1px solid $neutral-8;
-        }
-        .test-step-view-container {
```

Comment:
These changes look more extreme in the diff than they are:

I simply moved the test-step-view-container styles to the new styles file and test-step-view no longer rely on that hierarchy (verified that these are the only styles that apply.